examples/prisma-axiom/index.ts

```diff
@@ -1,9 +1,12 @@
 import withAxiom from '../../src/axiom';
 import { PrismaClient } from '@prisma/client';
+const express = require('express')
```

Comment:
Can we split `index.ts` and `main.ts` into `script` and `express` example folders

Reply:
done.